src/main/java/core/basesyntax/strategy/OperationBalancePerformer.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class OperationBalancePerformer implements OperationPerformer {
+    @Override
+    public void perform(FruitTransaction transaction, Storage storage) {

```

Comment:
we can access storage directly
```suggestion
    public void perform(FruitTransaction transaction) {

```